<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cifr AES ECB</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/document_encrypt.png</normaloff>:/new/prefix1/document_encrypt.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string>08 72 8b 81 99 a4 7e e8 45 38 38 42 20 ac 05 3f e2 a4 4a 05 f8 c8 b9 0f d2 2c d1 9c 09 53 8c 53 9a 79 ba 0c da 7f 29 8d ea 4c 78 7e  01 50 1c 28</string>
     </property>
     <property name="placeholderText">
      <string>Input hex (Len%16 == 0)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input len = %1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>00 01 02 03 04 05 06 07 08 09 0A 0B 0C 0D 0E 0F</string>
     </property>
     <property name="placeholderText">
      <string>Key hex 16 byte</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Key len = %1</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Encrypt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Decrypt</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>Clear Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>Clear Output</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Output hex</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
